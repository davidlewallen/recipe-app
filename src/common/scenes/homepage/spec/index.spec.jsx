import React from 'react';
import { shallow } from 'enzyme';

import Homepage from '../components';

describe('Homepage', () => {
  it('should render and match snapshot', () => {
    const wrapper = shallow(<Homepage />);
    expect(wrapper.find('Grid').exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
