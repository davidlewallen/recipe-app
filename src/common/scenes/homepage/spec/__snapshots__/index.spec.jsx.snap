// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage should render and match snapshot 1`] = `
<Grid
  bsClass="container"
  className="homepage"
  componentClass="div"
  fluid={false}
>
  <Row
    bsClass="row"
    componentClass="div"
  >
    <Col
      bsClass="col"
      className="align-center"
      componentClass="div"
      md={10}
      mdOffset={1}
      xs={12}
    >
      <div
        className="center-vertically"
      >
        <span>
          My Saved Recipes
        </span>
        <Link
          replace={false}
          to="/account/login"
        >
          <Button
            active={false}
            block={true}
            bsClass="btn"
            bsStyle="primary"
            disabled={false}
          >
            Login
          </Button>
        </Link>
      </div>
    </Col>
  </Row>
</Grid>
`;
